lms/djangoapps/instructor/views/api.py

```diff
@@ -1125,6 +1145,8 @@ def list_course_problems(request, course_id):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+@require_post_params(group_id="id of group to rename")
+@require_post_params(group_name="the new name of the group query with id group_id")
```

Comment:
You may be able to do that, but I find it easier to read w/ separate statements.

It will also diff more cleanly when changes are made later.